test/integration/02-user-wallet/02-receive-onchain.spec.ts

```diff
@@ -507,6 +508,43 @@ describe("UserWallet - On chain", () => {
   })
 })
 
+describe("BriaService", () => {
```

Comment:
should we create a new file for this?

Reply:
yes please

Reply:
Done https://github.com/GaloyMoney/galoy/pull/2596/commits/6d94f8babba07fd50dad32dd3b5086e9e8f45ea5